tests/end-to-end/simple/App.java

```diff
@@ -14,201 +12,137 @@
-    map.put(secondaryKey, value);
-    inner.put(primaryKey, map);
-    return this;
-  }
-
-  public CfnMapping get() {
-    return CfnMapping.Builder.create(this.scope, this.name).mapping(this.inner).build();
-  }
-}
+    public Object getQueueArn() {
+        return this.queueArn;
+    }
 
+    public Object getIsLarge() {
+        return this.isLarge;
+    }
+
+    public SimpleStack(final Construct scope, final String id) {
+        super(scope, id, null);
+    }
+
+    public SimpleStack(final Construct scope, final String id, final StackProps props) {
+        this(scope, id, props, null, null);
+    }
+
+    public SimpleStack(final Construct scope, final String id, final StackProps props,
+            String bucketNamePrefix,
+            String logDestinationBucketName) {
+        super(scope, id, props);
+
+        bucketNamePrefix = Optional.ofNullable(bucketNamePrefix).isPresent() ? bucketNamePrefix
+                : "bucket";
+        logDestinationBucketName = Optional.ofNullable(logDestinationBucketName).isPresent()
+                ? logDestinationBucketName
+                : CfnParameter.Builder.create(this, "LogDestinationBucketName")
+                        .type("AWS::SSM::Parameter::Value<String>")
+                        .defaultValue("/logging/bucket/name")
+                        .build()
+                        .getValueAsString();
+
+        // Mappings
+        final CfnMapping booleans = new CfnMapping(this, "booleans");
+        booleans.setValue("True", "true", true);
+        booleans.setValue("False", "false", false);
+
+        final CfnMapping lists = new CfnMapping(this, "lists");
+        lists.setValue("Candidates", "Empty", Arrays.asList(""));
+        lists.setValue("Candidates", "Singleton", Arrays.asList("One"));
+        lists.setValue("Candidates", "Pair", Arrays.asList("One", "Two"));
+
+        final CfnMapping numbers = new CfnMapping(this, "numbers");
+        numbers.setValue("Prime", "Eleven", 11);
+        numbers.setValue("Prime", "Thirteen", 13);
+        numbers.setValue("Prime", "Seventeen", 17);
+
+        final CfnMapping strings = new CfnMapping(this, "strings");
+        strings.setValue("Foos", "Foo1", "Foo1");
+        strings.setValue("Foos", "Foo2", "Foo2");
+        strings.setValue("Bars", "Bar", "Bar");
+
+        final CfnMapping table = new CfnMapping(this, "table");
+        table.setValue("Values", "Boolean", true);
+        table.setValue("Values", "Float", 3.14);
+        table.setValue("Values", "List", Arrays.asList("1", "2", "3"));
+        table.setValue("Values", "Number", 42);
+        table.setValue("Values", "String", "Baz");
+
+        Boolean isUs = Fn.select(0, Arrays.asList(this.getRegion().split("-"))).equals("us");
+        Boolean isUsEast1 = this.getRegion().equals("us-east-1");
+        Boolean isLargeRegion = isUsEast1;
+
+        CfnQueue queue = CfnQueue.Builder.create(this, "Queue")
+                .delaySeconds(42.1337)
```

Comment:
nit: My linter wasn't a major fan of the double white space on these resources. Not a blocker if we want to ship anyway imo

Reply:
How many spaces did your linter want here? I'm using the `Extension Pack for Java` plugin and I followed the spacing it automatically corrected to on save.

Reply:
it wants even more indentation 😅. I say we draw the line here and what you have is fine